### DISCUSSION TITLE
Time for another release?

### DISCUSSION THREAD

**Discussion by liu-song-6** [dynup/kpatch#1214] [General]
Hi folks, 

With the recent changes, we plan to ship another version of kpatch and kpatch-build to our fleet. Do we have plan to cut another release? This would make the process easier. 

Thanks,
Song

**Comment by jpoimboe** [comment]
No objections from me.  Any volunteers? ;-)

**Reply by yhcote** [reply]
I can take a shot at it.

**Reply by yhcote** [reply]
v0.9.4 was released

**Comment by liu-song-6** [comment]
Thanks a lot!

**Comment by liu-song-6** [comment]
Hi folks,

It is about 6 months since 0.9.5, shall we do another release?

Thanks!
